src/ControlSystem/WriteData.hpp

```diff
@@ -33,6 +33,7 @@ namespace control_system {
  * - d2tLambda
  * - ControlError
  * - dtControlError
+ * - Timescale
```

Comment:
Technically there's only one timescale that matters and that's the damping timescale. However, we often talk about the update timescale, measurement timescale, expiration time, etc. which are all based off the damping timescale. So that could have been kind of confusing. I changed the name to be `DampingTimescale`.

And there's one damping timescale for each component of a function of time, but since we write one file for each component, that's already taken care of.

Reply:
Okay, sounds good. Thanks for clarifying!